### PR TITLE
feat:Create and edit ecosystems

### PR DESCRIPTION
**Pull request by pranalidhanavade**
- Design create ecosystems modal.
- Design edit ecosystems modal.
- API integration for create ecosystems.

### CODE REVIEW

File: src/api/ecosystems.ts [hunk 1]
```diff
@@ -0,0 +1,62 @@
+import {  axiosPost, axiosPut } from "../services/apiRequests"
+import { apiRoutes } from "../config/apiRoutes";
+import { getFromLocalStorage } from "./Auth";
+import { storageKeys } from "../config/CommonConstant";
+
+export const createEcosystems = async (data: object) => {
```
Comment (vivekayanworks) [inline src/api/ecosystems.ts:6]: @pranalidhanavade can we not define an interface for data here instead of assigning a generic datatype as object.

**Review by vivekayanworks — CHANGES_REQUESTED** [review]

File: src/app/SideBar.astro [hunk 2]
```diff
@@ -94,6 +94,63 @@ import { pathRoutes } from "../config/pathRoutes";
...
+								xmlns="http://www.w3.org/2000/svg"
+								aria-hidden="true"
+							>
+								<path
+									clip-rule="evenodd"
+									fill-rule="evenodd"
+									d="M.99 5.24A2.25 2.25 0 013.25 3h13.5A2.25 2.25 0 0119 5.25l.01 9.5A2.25 2.25 0 0116.76 17H3.26A2.267 2.267 0 011 14.74l-.01-9.5zm8.26 9.52v-.625a.75.75 0 00-.75-.75H3.25a.75.75 0 00-.75.75v.615c0 .414.336.75.75.75h5.373a.75.75 0 00.627-.74zm1.5 0a.75.75 0 00.627.74h5.373a.75.75 0 00.75-.75v-.615a.75.75 0 00-.75-.75H11.5a.75.75 0 00-.75.75v.625zm6.75-3.63v-.625a.75.75 0 00-.75-.75H11.5a.75.75 0 00-.75.75v.625c0 .414.336.75.75.75h5.25a.75.75 0 00.75-.75zm-8.25 0v-.625a.75.75 0 00-.75-.75H3.25a.75.75 0 00-.75.75v.625c0 .414.336.75.75.75H8.5a.75.75 0 00.75-.75zM17.5 7.5v-.625a.75.75 0 00-.75-.75H11.5a.75.75 0 00-.75.75V7.5c0 .414.336.75.75.75h5.25a.75.75 0 00.75-.75zm-8.25 0v-.625a.75.75 0 00-.75-.75H3.25a.75.75 0 00-.75.75V7.5c0 .414.336.75.75.75H8.5a.75.75 0 00.75-.75z"
+								></path>
+							</svg>
+							<span class="ml-3" sidebar-toggle-item>Ecosystems</span>
```
[the remark this answers is not in the document]
Comment (pranalidhanavade) [reply src/app/SideBar.astro]: Resolved

File: src/config/pathRoutes.ts [hunk 3]
```diff
@@ -35,14 +35,15 @@ export const pathRoutes = {
...
-    // },
+    ecosystems: {
+        root: '/ecosystems',
+        dashboard:'/ecosystems/dashboard'
+        // frameworks: '/ecosystems/frameworks',
+        // members: '/ecosystems/members',
+        // registries: '/ecosystems/registries',
+        // users: '/organizations/users',
+        // credentials: '/organizations/credentials'
+    },
```
[the remark this answers is not in the document]
Comment (pranalidhanavade) [reply src/config/pathRoutes.ts:41]: Resolved

File: src/api/ecosystems.ts (hunk 1 above)
Comment (pranalidhanavade) [reply src/api/ecosystems.ts]: Added interface for that.

**Pull request merged**